packages/core/src/validate/index.ts

```diff
@@ -55,6 +55,22 @@ export const validateRelease = (releaseJson: MetaplexFileReleaseJsonMetadata) =>
     jsonToValidate.image = jsonToValidate.image?.fileName;
   }
 
+  // Support the replacing of string URIs with a MetaplexFile instance
```

Comment:
Can I get a quick summary of why this new code is required for Nodejs 18 support?

Reply:
So I actually think this particular issue was the result of adding uri to into the schema. I didn't run the app at that time, but I knew I would before cutting an actual new release. So now it came up.